index.html

```diff
@@ -1140,6 +1140,27 @@ <h4>Timing attacks</h4>
       This attack is mitigated by [[[#data-minimization]]], [[[#rate-limiting-change-notifications]]],
       and [[[#same-origin-restriction]]].
     </p>
+    <h4>Cross-site covert channel</h4>
+    <p>
+      In computer security a covert channel creates a capability to transfer information between processes
+      that are not supposed to be allowed to communicate. In modern multi-process web engines in the generic
+      case each window or tab resides in its own process (documents that have the [=same origin=] or sites that
+      have the [=same site=] typically share the same process). Using this API it may be possible to create a
+      cross-site covert channel C where a site A on one tab first writes to the channel C after having
```

Comment:
I am not sure "write" is the right word, it is more like broadcasting

Reply:
Fixed in https://github.com/w3c/compute-pressure/pull/219/commits/9e8cd0fbda86993a09bfb6f424e27522e30cc73b